include/core/parameters.h

```diff
@@ -1616,22 +1616,50 @@ namespace Parameters
     parse_parameters(ParameterHandler &prm);
   };
 
-
   /**
-   * @brief Return the tensor of entry @p entry_string. If the entry is specified
-   * in the parameter file, then the changed value is returned, otherwise the
-   * default value is returned. If the entry is not equivalent to a Tensor<1,3>,
-   * an error will be thrown. This function can be use for Point<3> variables.
+   * @brief Return the tensor of entry @p entry_string. If the entry is
+   * specified in the parameter file, then the changed value is returned,
+   * otherwise the default value is returned. It can be used for 2 ou 3
+   * dimensions parameters that require a tensor of dimension 3 in any case.
+   * If the entry do not have 2 or 3 values in the list, an error is throw.
```

Comment:
```suggestion
   * @brief Return the tensor of entry @p entry_string. If the entry is
   * specified in the parameter file, then the changed value is returned,
   * otherwise the default value is returned. It can be used for 2 or 3
   * dimensions parameters that require a tensor of dimension 3 in any case.
   * If the entry does not have 2 or 3 values in the list, an error is thrown.
```